core/templates/domain/story/editable-story-backend-api.service.spec.ts

```diff
@@ -16,45 +16,39 @@
  * @fileoverview Unit tests for EditableStoryBackendApiService.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// the code corresponding to the spec is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
-
-import { TranslatorProviderForTests } from 'tests/test.extras';
-
-require('domain/story/editable-story-backend-api.service.ts');
-require('services/csrf-token.service.ts');
-
-describe('Editable story backend API service', function() {
-  var EditableStoryBackendApiService = null;
-  var sampleDataResults = null;
-  var $httpBackend = null;
-  var CsrfService = null;
-
-  beforeEach(angular.mock.module('oppia'));
-  beforeEach(
-    angular.mock.module('oppia', TranslatorProviderForTests));
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
-    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
-      $provide.value(key, value);
-    }
-  }));
-
-  beforeEach(angular.mock.inject(function($injector, $q) {
-    EditableStoryBackendApiService = $injector.get(
-      'EditableStoryBackendApiService');
-    $httpBackend = $injector.get('$httpBackend');
-    CsrfService = $injector.get('CsrfTokenService');
-
-    spyOn(CsrfService, 'getTokenAsync').and.callFake(function() {
-      var deferred = $q.defer();
-      deferred.resolve('sample-csrf-token');
-      return deferred.promise;
+import { EditableStoryBackendApiService } from
+  'domain/story/editable-story-backend-api.service';
+import { CsrfTokenService } from 'services/csrf-token.service';
+
+import { HttpClientTestingModule, HttpTestingController } from
+  '@angular/common/http/testing';
+import { TestBed, fakeAsync, flushMicrotasks } from
+  '@angular/core/testing';
```

Comment:
Maybe move these above the other two imports.

Reply:
done